config/initializers/rollbar.rb

```diff
@@ -1,5 +1,5 @@
 Rollbar.configure do |config|
-  code_version = ENV.fetch('GIT_REV', nil)
+  code_version = ENV.fetch('GIT_REV', `git log --format=format:%H | head -n 1`.rstrip)
```

Comment:
See https://github.com/davidrunger/david_runger/pull/5063 for an improvement on this.